lang/cs/Org.Apache.REEF.Network/Group/Config/GroupCommConfigurationOptions.cs

```diff
@@ -44,7 +44,7 @@ public class Timeout : Name<int>
         {
         }
 
-        [NamedParameter("Retry times", defaultValue: "5")]
+        [NamedParameter("Retry times", defaultValue: "500")]
         public class RetryCount : Name<int>
```

Comment:
This retry count is used in waiting for registration. When we have over 100 nodes, 5 times is far from enough. 
We don't want set it too be too low in case some nodes are just slow, if we simply throw an exception, that is not right. 
We don't want it to try endlessly in case some node is really dead, we should come out with exception. 
We want it to return as soon as all nodes in the group are registered, So increasing retry count is better then increasing sleep time. 
Dhruv and I discussed about it based on the cluster testing, this number would give 250 seconds, about 4 min waiting time before throwing exception.